setup.py

```diff
@@ -25,9 +25,9 @@
 # name of Interfacing API
 APPNAME = "pyprecice"
 # this version should be in sync with the latest supported preCICE version
-precice_version = version.Version("2.1.0")  # todo: should be replaced with precice.get_version(), if possible or we should add an assertion that makes sure that the version of preCICE is actually supported
+precice_version = version.Version("2.1.1")  # todo: should be replaced with precice.get_version(), if possible or we should add an assertion that makes sure that the version of preCICE is actually supported
 # this version number may be increased, if changes for the bindings are required
-bindings_version = version.Version("1")
+bindings_version = version.Version("1rc1")
```

Comment:
This has to be changed to `"1"` for the "real" release. We do not want to merge pre-releases into master.

If you want to create a release candidate: Only hit the "Publish Release" button under ["Draft a new Release"](https://github.com/precice/python-bindings/releases/new). Don't forget to check "This is a pre-release". This will automatically create a new tag. The creation of the tag will trigger the github actions workflow and publish the release candidate [to PyPI](https://pypi.org/project/pyprecice/#history).